test/src/org/omegat/gui/main/MainWindowMenuTest.java

```diff
@@ -84,7 +97,63 @@ public void testMenuActions() throws Exception {
             }
         }
         assertTrue("menu items not found", count > 30);
-        assertTrue("There is action handlers in MainWindow which doesn't used in menu: " + existsMethods.keySet(),
-                existsMethods.isEmpty());
+        assertTrue("There is action handlers in MainWindow which doesn't used in menu: "
+                + existsMethods.keySet(), existsMethods.isEmpty());
+    }
+
+    @Test
+    public void testMenuActions_invokeActions() throws Exception {
+        Pattern pattern = Pattern.compile("getMainMenu\\(\\).invokeAction\\(\\s*\"([^\"]+)\"\\s*,");
+        final String[] target = {"src"};
+        processSourceContent(target, (path, chars) -> {
+            Matcher matcher = pattern.matcher(chars);
+            while (matcher.find()) {
+                String actionMethodName = matcher.group(1) + "ActionPerformed";
+                Method method = null;
+                try {
+                    method = MainWindowMenuHandler.class.getMethod(actionMethodName);
+                } catch (NoSuchMethodException ignore) {
+                    // See if the method accepts a modifier key argument.
+                    try {
+                        method = MainWindowMenuHandler.class.getMethod(actionMethodName, Integer.TYPE);
+                    } catch (NoSuchMethodException ex) {
+                        assertNotNull("Action method \"" + actionMethodName + "\" not defined for invoked from " + path,
+                                method);
+                    }
+                }
+            }
+        });
+    }
+
+    /**
+     * Process the text content of all .java files under /src. Also check
+     * DIRECTIONALITY_LEFT_TO_RIGHT_OVERRIDE for security reasons.
+     * And check invisible space character.
```

Comment:
Javadoc is not correct. I removed the feature which is new rule for master branch.